### ISSUE TITLE
wake word with porcupine

### ISSUE THREAD

**Issue by Khomotica** [synesthesiam/rhasspy-hassio-addon#123]
Hi,
i tried to use porcupine for best performances on the wake word process as promised but every time i fall on error :

> ERROR:PorcupineWakeListener:loading wake handle
Traceback (most recent call last):
  File "/home/pi/rhasspy/rhasspy/wake.py", line 805, in in_started
    self.load_handle()
  File "/home/pi/rhasspy/rhasspy/wake.py", line 875, in load_handle
    sensitivities=self.sensitivities,
  File "/home/pi/rhasspy/porcupine.py", line 116, in __init__
    raise self._PICOVOICE_STATUS_TO_EXCEPTION[status]('Initialization failed')
ValueError: Initialization failed

However when i switch to snowboy it's working

BTW : good job man for what you're doing specially for the guys who are a lot disappointed in the snips tragedy ;)

**Comment by synesthesiam** [comment]
Thanks!

This error is something I haven't come across before. Are you using a 64-bit Raspbian (`aarch64`) by chance?

**Comment by Khomotica** [comment]
Nop just the rpi 3 B+ (32 bit).

Other info which maybe it's relevant : i used a linux custom model from porcupine console (free one) and i don't know if it's the root cause.

**Comment by synesthesiam** [comment]
Ah, maybe my Porcupine libraries in Rhasspy are out of date.

**Comment by frkos** [comment]
But does it work when you use raspberry file,  not linux one?
https://github.com/Picovoice/porcupine/tree/master/resources/keyword_files/raspberrypi

**Comment by Khomotica** [comment]
the same with Terminator Rasp sample :

> ERROR:PorcupineWakeListener:loading wake handle
Traceback (most recent call last):
  File "/home/pi/rhasspy_dev/rhasspy/wake.py", line 798, in to_started
    self.load_handle()
  File "/home/pi/rhasspy_dev/rhasspy/wake.py", line 876, in load_handle
    sensitivities=self.sensitivities,
  File "/home/pi/rhasspy_dev/porcupine.py", line 116, in __init__
    raise self._PICOVOICE_STATUS_TO_EXCEPTION[status]('Initialization failed')

@synesthesiam : BTW this is why i proposed on roadmap discussion to include some tool to manage outadated lib/packages

**Comment by synesthesiam** [comment]
Rhasspy appears to be on the very latest version of Porcupine, so that's not it. Is there something special you're supposed to do for a custom wakeword?

**Comment by manju-rn** [comment]
Not sure if this is relevant to this issue, but I was not able to get a custom wake word called until I entered the word in the Dictonary.txt  When I tried to recreate the error, the log did not accurately report the error, only once or twice the issue was clearly reported as the word not available in Dictionary.

As a snips victim, let me also add my appreciation for Rhasspy  - great product man. I was thrilled when I saw that I have an option to retain my existent intent or directly call the HA events.  Best of all, I do not have to compromise my 'conversation' - I could either call them from Rhasspy or retain it as it is  - I was frustrated when I implemented Almond.
